data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all parts of the game have been implemented yet."`
```

If you'd rather phrase it as "parts," this would be the simpler and less awkward option.